### ISSUE TITLE
Vrf parser error on nxos

### ISSUE THREAD

**Issue by mike-mosher** [CiscoTestAutomation/genieparser#38]
Running nxos9000v on ViRL, and get error for ["show running-config vrf <vrf> | sec '^vrf'"](https://pubhub.devnetcloud.com/media/genie-feature-browser/docs/#/parsers/show%20running-config%20vrf%20%7Bvrf%7D%20%7C%20sec%20'%5Evrf') parser:

Command:
```
genie parse "show running-config vrf test-vrf | sec '^vrf'" --testbed-file testbed.yaml --devices device1 --output output/
```

Error:
```
Issue with the parser show running-config vrf test-vrf | sec '^vrf'

Traceback (most recent call last):
  File "src/genie/cli/commands/parser.py", line 276, in genie.cli.commands.parser.ParserCommand.parse
  File "src/genie/conf/base/device.py", line 498, in genie.conf.base.device.Device.parse
  File "src/genie/metaparser/_metaparser.py", line 271, in genie.metaparser._metaparser.MetaParser.parse
TypeError: cli() got an unexpected keyword argument 'feature'
```

And in python:
```
from genie.conf import Genie

vrf_name = 'test-vrf'
vrf_parser = f"show running-config vrf {vrf_name} | sec '^vrf'"

testbed = Genie.init('testbed.yaml')
device = testbed.devices['device1']
device.connect()

running_config = device.parse(vrf_parser)
print(json.dumps(running_config, indent=4))
```

Error:
```
running_config = device.parse(vrf_parser)
  File "src/genie/conf/base/device.py", line 498, in genie.conf.base.device.Device.parse
  File "src/genie/metaparser/_metaparser.py", line 271, in genie.metaparser._metaparser.MetaParser.parse
TypeError: cli() got an unexpected keyword argument 'feature'
```

testbed.yaml:
```
testbed:
  name: testbed
devices:
  device1:
    os: nxos
    type: nxosv
    connections:
      defaults:
        class: unicon.Unicon
      telnet:
        protocol: telnet
        ip: 172.16.1.191
        username: "cisco"
        password: "cisco"
    custom:
      abstraction:
        order: [os, type]
```

**Comment by tahigash** [comment]
@mike-mosher thank you for reporting this. We will look into this.

**Comment by thaixuann** [comment]
Hello @mike-mosher 
We fixed the issue and genie package v19.9.1b1 has been released with the fix.
Could you please run the following command and try again to see if it works?

pip install genie.libs.parser --upgrade --pre

Thanks,
--Thai.

**Comment by jeaubin** [comment]
Closing, let us know if any other issue.
